pkg/server/backend_manager.go

```diff
@@ -234,7 +234,7 @@ func (s *DefaultBackendStorage) RemoveBackend(identifier string, idType pkgagent
 		}
 	}
 	if !found {
-		klog.V(1).InfoS("Cannot find connection for agent in backends", "connection", conn, "agentID", agentID)
+		klog.V(1).InfoS("Cannot find connection for agent in backends", "connection", conn, "identifier", identifier)
```

Comment:
Could we rephrase this to not mention agent? Eg. "Could not find connection matching identifier to remove"?
Also could you fix the log message on line 205 to indicate failed to remove not failed to add?